revenuecatui/src/androidMain/kotlin/com/revenuecat/purchases/kmp/ui/revenuecatui/CustomerCenter.kt

```diff
@@ -0,0 +1,15 @@
+package com.revenuecat.purchases.kmp.ui.revenuecatui
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import com.revenuecat.purchases.ui.revenuecatui.customercenter.CustomerCenter as RcCustomerCenter
+
+@Composable
+public actual fun CustomerCenter(
+    onDismiss: () -> Unit
```

Comment:
It's a good practice to allow passing a modifier with a default to `Modifier` in composables

```
@Composable
public actual fun CustomerCenter(
    modifier: Modifier = Modifier, // Allow customization
    onDismiss: () -> Unit
): Unit =
    RcCustomerCenter(
        modifier = modifier.fillMaxSize(), // Use passed modifier
        onDismiss = onDismiss
    )
```